src/Filter/AbstractDateFilter.php

```diff
@@ -119,26 +124,41 @@ public function getRenderSettings()
         ]];
     }
 
+    abstract protected function applyTypeIsLessEqual(ProxyQueryInterface $queryBuilder, string $field, array $data);
+
+    abstract protected function applyTypeIsGreaterThan(ProxyQueryInterface $queryBuilder, string $field, array $data);
+
+    abstract protected function applyTypeIsEqual(ProxyQueryInterface $queryBuilder, string $field, array $data);
```

Comment:
This looks like a BC-break for extending classes

Reply:
It is a little bit tricky because these methods are called [within the own class](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/3.x/src/Filter/AbstractDateFilter.php#L58), so they should have been added in the extending classes, but it's true that someone may have not... so should I just add them as `@method`?

Reply:
Oh I didn't notice. I think it's fine actually, nevermind my comment.